### ISSUE TITLE
Disable "check word meaning" and "annotate selected text" when nothing is selected

### ISSUE THREAD

**Issue by johnfactotum** [babluboy/bookworm#267]
In the context menu, "check word meaning" and "annotate selected text" should be disabled or hidden when nothing is selected.

**Comment by babluboy** [comment]
@johnfactotum Good point and agree with the same - will try to implement in this release.

**Comment by babluboy** [comment]
Pushed a partial [fix](https://github.com/babluboy/bookworm/commit/c675e0f8a4c273c5e87fa12b707f1df0662646b6) for this issue.

Setting the menu items as disabled on the webkit context menu removes the menu, so added a condition to only execute the word meaning and annotation only if a selection is made. If no selection is done then selecting word meaning and annotation in the context menu does not do anything.

Will keep this open to try and make the items disabled rather than enabled when no selection is done.
